airbyte-integrations/connectors/source-facebook-marketing/unit_tests/resource/http/response/ad_sets.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Could you please update these files so the data looks exactly like real responses?

Reply:
updated, we have these records in expected records file